xml/System.Net.Http/HttpClient.xml

```diff
@@ -25,7 +25,7 @@
 ## Remarks  
  The <xref:System.Net.Http.HttpClient> class instance acts as a session to send HTTP requests. An <xref:System.Net.Http.HttpClient> instance is a collection of settings applied to all requests executed by that instance. In addition, every <xref:System.Net.Http.HttpClient> instance uses its own connection pool, isolating its requests from requests executed by other <xref:System.Net.Http.HttpClient> instances.  
   
- The <xref:System.Net.Http.HttpClient> also acts as a base class for more specific HTTP clients. An example would be a FacebookHttpClient providing additional methods specific to a Facebook web service (a GetFriends method, for instance).  
+ The <xref:System.Net.Http.HttpClient> also acts as a base class for more specific HTTP clients. An example would be a FacebookHttpClient providing additional methods specific to a Facebook web service (a GetFriends method, for instance). Derived classes should not override the virtual methods on the class. Use the constructor overload accepting <xref:System.Net.Http.HttpMessageHandler> to configure any pre- or post-request processing instead.
```

Comment:
LGTM. Wondering if it would be better to say "Use a constructor overload that accepts" since there's more than one though.

/cc @stevewhims

Reply:
The "instead" works well to flow the thought from one sentence to the next, but I'd put the "instead" at the start of the 2nd sentence. Then the reader doesn't need to wait till the end of the sentence to parse that connection. So: "Instead, use the constructor overload that accepts ...".